pokemon/convert.py

```diff
@@ -66,23 +57,21 @@ def convert_image_to_ascii(image, new_width):
     len_pixels_to_chars = len(pixels_to_chars)
 
     image_ascii = [pixels_to_chars[index: index + new_width*2] for index in
-            xrange(0, len_pixels_to_chars, new_width*2)]
-
+            range(0, len_pixels_to_chars, new_width*2)]
     return "\n".join(image_ascii)
 
-def handle_image_conversion(image_filepath,new_width=100):
+def handle_image_conversion(image_filepath,new_width):
     image = None
     try:
         image = Image.open(image_filepath)
     except:
         print("Unable to open image file {image_filepath}.".format(image_filepath=image_filepath))
         print(e)
         return
-
     image_ascii = convert_image_to_ascii(image,new_width)
-    print(image_ascii)
```

Comment:
Where is the print now?

Reply:
Ah, I thought I added that myself to verify the ascii images were correct. Adding back in.